doc/modules/compiler/utils.rst

```diff
@@ -45,6 +45,30 @@ Their job is three-fold:
   supplied to either pass on construction to encode this metadata. If not set,
   the default ``xyz`` order is used.
 
+LowerToMuxBuiltinsPass
+----------------------
+
+Replaces calls to the language implementation's builtin functions with
+alternative sequences of instructions involving ComputeMux builtins.
+
+Targets **must** lower their language builtins to ComputeMux builtins in order
+for other ComputeMux compiler passes to corectly recognise the program
+semantics. This is because the compiler passes provided by the oneAPI
+construction kit only understand the semantics ComputeMux builtins, and thus
```

Comment:
sentence does not scan right

Reply:
aye, you're right. I've tried to sort it out by rewriting this paragraph.